common/src/main/java/io/quarkus/ts/openshift/common/deploy/UsingQuarkusPluginDeploymentStrategy.java

```diff
@@ -121,15 +117,13 @@ private void withMavenRepositoryLocalIfSet(List<String> args) {
         }
     }
 
-    private void withQuarkusBom(List<String> args) {
-        String groupId = System.getProperty(QUARKUS_PLATFORM_GROUP_ID);
-        if (groupId != null) {
-            args.add(withProperty(QUARKUS_PLATFORM_GROUP_ID, groupId));
-        }
-
-        String artifactId = System.getProperty(QUARKUS_PLATFORM_ARTIFACT_ID);
-        if (artifactId != null) {
-            args.add(withProperty(QUARKUS_PLATFORM_ARTIFACT_ID, artifactId));
+    private void withQuarkusProperties(List<String> args) {
```

Comment:
This looks a bit more reader friendly
```java
System.getProperties().entrySet().stream()
      .filter(entry -> StringUtils.startsWith((String)entry.getKey(), QUARKUS_PROPERTY_PREFIX))
      .filter(entry -> StringUtils.isNotEmpty((String)entry.getValue()))
      .forEach(entry -> args.add(withProperty((String)entry.getKey(), (String)entry.getValue())));
```
Otherwise LGTM

Reply:
Changed to use predicates as you suggested. Let me know if now it looks better.